src/theme/styled.d.ts

```diff
@@ -128,10 +128,16 @@ declare module 'styled-components/macro' {
 
     // media queries
     mediaWidth: {
-      upToExtraSmall: ThemedCssFunction<DefaultTheme>
-      upToSmall: ThemedCssFunction<DefaultTheme>
-      upToMedium: ThemedCssFunction<DefaultTheme>
-      upToLarge: ThemedCssFunction<DefaultTheme>
+      deprecated_upToExtraSmall: ThemedCssFunction<DefaultTheme>
```

Comment:
what is this type? is it not just a number?

Reply:
we can deprecate the entire `mediaWidths` and make a new one called `breakpoints`

Reply:
turns out its doing some css snippet magic